libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -78,6 +80,7 @@ pub use error::BiomeRestResourceManagerBuilderError;
 ))]
 use self::actix::{
     login::make_login_route,
+    register::make_register_route,
```

Comment:
It doesn't look like this should be added in this commit, but one of the previous commit. Also, it looks like the `make_register_route` is the only one that should be behind the `biome-refresh-tokens` feature. It's also added twice within this file.

Reply:
Fixed commits